src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -764,6 +768,15 @@ public function deprecateParameter(string $name, string $package, string $versio
         $this->parameterBag->deprecate($name, $package, $version, $message);
     }
 
+    public function nonEmptyParameter(string $name, string $message = 'A non-empty value for the parameter "%s" is required.'): void
+    {
```

Comment:
Should we check if name is an empty string here?

Reply:
Just to clarify, it's the parameter value that needs to be non-empty, not the name. Currently the parameter name isn't being validated anywhere. If you feel it's worth addressing, it could be something to consider for another PR. Personally I don’t think it's necessary, but I'm open to your thoughts!